modules/programs/k9s.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.k9s;
+  yamlFormat = pkgs.formats.yaml { };
+in {
+  meta.maintainers = [ hm.maintainers.katexochen ];
+
+  options.programs.k9s = {
+    enable =
+      mkEnableOption "k9s - Kubernetes CLI To Manage Your Clusters In Style!";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.k9s;
+      defaultText = literalExpression "pkgs.k9s";
+      description = "The k9s package to install.";
+    };
+
+    config = mkOption {
+      type = yamlFormat.type;
+      default = { };
+      description = ''
+        Configuration written to
+        <filename>$XDG_CONFIG_HOME/k9s/config.yml</filename>. See
+        <link xlink:href="https://k9scli.io/topics/config/"/>
+        for supported values.
+      '';
+      example = literalExpression ''
+        k9s = {
+          config = {
```

Comment:
Should this really be there? As far as I can tell from https://k9scli.io/topics/config there is no `config` key.
```suggestion
```